subjects/office_worker/README.md

```diff
@@ -0,0 +1,58 @@
+## office_worker
+
+### Instructions
+
+Create a structure `OfficeWorker` having the following public fields:
```

Comment:
Running locally `npx prettier --write .` is giving me an extra new line here

Reply:
done